utils/helpers.ts

```diff
@@ -184,3 +184,74 @@ export const cleanQuickstartName = (str: string): string =>
     .replace(/\s+/g, '-')
     .replace(/-+/, '-')
     .replace(/[^a-z0-9-]/g, '');
+
+/**
+ * Gets the unique base quickstart directory from a given file path.
+ * e.g. filePath: 'quickstarts/python/aiohttp/alerts/ApdexScore.yml' + targetChild: 'alerts' = 'python/aiohttp'.
+ * @param {String} filePath - Full file path of a file in a quickstart.
+ * @param {String} targetChild - Node in file path that should be preceded by a base quickstart directory.
+ * @return {String} Node in file path of the quickstart.
```

Comment:
It looks like we're omitting the type information in the JSDoc comments in the other files in this PR. We might want to keep this file consistent:

```suggestion
 * @param filePath - Full file path of a file in a quickstart.
 * @param [targetChild] - Node in file path that should be preceded by a base quickstart directory.
 * @return Node in file path of the quickstart.
```

Reply:
Done!